docs/src/main/paradox/server.md

```diff
@@ -164,25 +164,8 @@ akka.http.server.preview.enable-http2 = on
 To run the server with HTTP/2 enabled correctly, you will likely have to configure the Jetty ALPN
 agent as described @extref[in the Akka HTTP documentation](akka-http:server-side/http2.html#application-layer-protocol-negotiation-alpn-):
 
-sbt
-:   @@snip [build.sbt]($root$/../plugin-tester-scala/build.sbt) { #alpn }
-
-After that you can run it as usual:
-
-sbt
-:   ```
-runMain io.grpc.examples.helloworld.GreeterServer
-```
-
-Gradle
-:   ```
-./gradlew run
-```
-
-Maven
-:   ```
-mvn akka-grpc:generate compile exec:java -Dexec.mainClass=io.grpc.examples.helloworld.GreeterClient
-```
+See the detailed chapters on @ref[sbt](sbt.md#starting-your-akka-grpc-server-from-sbt), @ref[Gradle](gradle.md#starting-your-akka-grpc-server-from-gradle)
```

Comment:
I agree adding this paragraph here.

I wonder if we should keep the 'simple case' sbt/gradle/maven snippets though, to make this section read more like a tutorial when you see it for the first time (while providing the links to dig deeper if you need to)?

Reply:
The reason I didn't do that is because there isn't really a simple case, not even for sbt where you still have to add stuff to both plugin.sbt and build.sbt before you can actually run anything, so that is three code blocks that needs to coexist with other code blocks for the other two build tools (which are also not simple steps for maven and I'm  guessing the same will be true for gradle?)